src/frontend/src/components/EditProfile.svelte

```diff
@@ -0,0 +1,53 @@
+<script>
+  import { onMount } from "svelte";
+  import { actor, crypto_service, init_nostr_structures} from "../store/auth";
+  import ProfileForm from "./ProfileForm.svelte";
+
+  let private_key;
+  let pk;
+  let username = "";
+  let about = "";
+  let avatar_url = "";
+  let loading = false;
+
+  const getProfile = async () => {
```

Comment:
nemám to odskúšané, ale je toto nutné? ak sa nemýlim, at this point je ten profil už načítaný v exportovanej premennej `nostric_user` v `auth`. Stačí si ho sem zobrať cez metódu `get_profile` a rozhodiť do tých premenných.